buildout.cfg

```diff
@@ -1,10 +1,24 @@
 [buildout]
-extends = http://download.zope.org/Zope2/index/2.13.19/versions.cfg
-parts = test
+extensions =
+    mr.developer
+extends = 
+    https://raw.githubusercontent.com/zopefoundation/Zope/master/sources.cfg
+    https://raw.githubusercontent.com/zopefoundation/Zope/master/versions-prod.cfg
 develop = .
+parts = 
+    test
+    tox
+auto-checkout =
+    Zope2
```

Comment:
4.0a4, the latest release, still thows this traceback, which is fixed on master:

```python
Traceback (most recent call last):
  File "/Users/jens/src/five.localsitemanager/src/five/localsitemanager/__init__.py", line 21, in <module>
    from five.localsitemanager.registry import PersistentComponents
  File "/Users/jens/src/five.localsitemanager/src/five/localsitemanager/registry.py", line 23, in <module>
    from OFS.ObjectManager import ObjectManager
  File "/Users/jens/src/.eggs/Zope2-4.0a4-py3.6.egg/OFS/ObjectManager.py", line 26, in <module>
    from types import NoneType
ImportError: cannot import name 'NoneType'
```